dstk-web/src/routes/dashboard/index.tsx

```diff
@@ -1,62 +1,66 @@
-import { Disclosure } from '@headlessui/react'
-import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+import { Disclosure } from '@headlessui/react';
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline';
 
 const nav = 'Never gonna give you up 🎶';
 
 export const Dashboard = () => {
-  return (
-    <div className='flex flex-col min-h-screen'>
-      {/* Navbar */}
-      <Disclosure as="header" className="bg-gray-800">
-        {({ open }) => (
-          <>
-            <div className="mx-auto max-w-7xl px-2 sm:px-6 lg:px-8">
-              <div className="relative flex h-16 items-center justify-between">
-                <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">
-                  {/* Mobile menu button*/}
-                  <Disclosure.Button className="relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white">
-                    {open ? (
-                      <XMarkIcon className="block h-6 w-6" aria-hidden="true" />
-                    ) : (
-                      <Bars3Icon className="block h-6 w-6" aria-hidden="true" />
-                    )}
-                  </Disclosure.Button>
-                </div>
+    return (
+        <div className='flex flex-col min-h-screen'>
+            {/* Navbar */}
+            <Disclosure as='header' className='bg-gray-800'>
+                {({ open }) => (
+                    <>
+                        <div className='mx-auto max-w-7xl px-2 sm:px-6 lg:px-8'>
+                            <div className='relative flex h-16 items-center justify-between'>
+                                <div className='absolute inset-y-0 left-0 flex items-center sm:hidden'>
+                                    {/* Mobile menu button*/}
+                                    <Disclosure.Button className='relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white'>
```

Comment:
How do we feel about line length?

Reply:
I have it set to 100, but I think className length overrides this.

Up to you, I don't really care but that's because I'm a Tailwind lover. I can break up long classNames to multiple lines if ya want.

Reply:
I'm so conditioned to PEP8 that anything over 80 characters makes my brain shut down. Thank youuuuu

Reply:
I feel ya. I think 80 line length + 4-space indents are the way to go in most cases cause I feel like vertical code is easier to digest to horizontal code.

Except for frontend projects (especially) using Tailwind. It kind of blows up your jsx as a side effect lol.